Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
```

Comment:
<p>latex does not show all the formulae in this NBreview page.</p><p>Also, do you want to give an explicit reference to the literature for this model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>LaTeX fixed and ref added, thanks.</p><p> </p>